src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/UserListRow.tsx

```diff
@@ -85,20 +72,7 @@ interface UserListRowProps {
 export const UserListRow = ({ playlistAccess }: UserListRowProps) => {
   const intl = useIntl();
 
-  const options = [
-    {
-      label: intl.formatMessage(messages.adminLabel),
-      key: PlaylistRole.ADMINISTRATOR,
-    },
-    {
-      label: intl.formatMessage(messages.studentLabel),
-      key: PlaylistRole.STUDENT,
-    },
-    {
-      label: intl.formatMessage(messages.instructorLabel),
-      key: PlaylistRole.TEACHER,
-    },
-  ];
+  const options = useMemo(() => userRoleOptions(intl), [intl]);
```

Comment:
IMO we don't need useMemo, the memoization will be probably more greedy that the init of the array.